src/Analyzers/CSharp/CodeFixes/UseInterpolatedString/CSharpUseInterpolatedStringCodeFixProvider.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Immutable;
+using System.Diagnostics.CodeAnalysis;
+using System.Composition;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.CodeActions;
+using Microsoft.CodeAnalysis.CodeFixes;
+using Microsoft.CodeAnalysis.UseInterpolatedString;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.Diagnostics;
+using Microsoft.CodeAnalysis.Editing;
+using Microsoft.CodeAnalysis.Host.Mef;
+using Microsoft.CodeAnalysis.LanguageService;
+using Microsoft.CodeAnalysis.Shared.Extensions;
+
+namespace Microsoft.CodeAnalysis.CSharp.UseInterpolatedString;
+
+[ExportCodeFixProvider(LanguageNames.CSharp, Name = PredefinedCodeFixProviderNames.UseInterpolatedString), Shared]
+internal partial class CSharpUseInterpolatedStringCodeFixProvider : AbstractUseInterpolatedStringCodeFixProvider
+{
+    [ImportingConstructor]
+    [Obsolete(MefConstruction.ImportingConstructorMessage, error: true)]
+    public CSharpUseInterpolatedStringCodeFixProvider()
```

Comment:
nit: can use primary constructors.

Note: `nits` are minor suggestions.  You are free to ignore them if you don't want to do them :)

Reply:
(just say you're ignoring it :))

Reply:
VS didn't seem to have an analyzer for this, so I had to fix this manually. Not sure if this is worth reporting, but I'll keep this thread open for now.